infra/Pos/Slotting/Error.hs

```diff
@@ -14,12 +14,17 @@ import           Pos.Exception       (cardanoExceptionFromException,
                                       cardanoExceptionToException)
 
 -- | Type aggregating run-time errors related to Slotting.
-data SlottingError = SEUnknownSlotStart !SlotId
+data SlottingError
+    = SEUnknownSlotStart !SlotId
+    | SEUnknownSlot
```

Comment:
`getSlotStartEmpatically` uses `SEUnknownSlotStart`, not `SEUnknownSlot` unless I am blind.